Gemfile

```diff
@@ -2,7 +2,7 @@
 
 git_source(:github) { |name| "https://github.com/#{name}.git" }
 source "https://rubygems.org"
-ruby "2.6.5"
+ruby "~> #{`cat .ruby-version`.strip}"
```

Comment:
In my private projects I generally keep this to pure Ruby, without shelling out:

```ruby
ruby "~> #{File.read('.ruby-version').strip}"
```

Reply:
I don't think we should have the `~>` as we want to make sure we have the same exact version both on Travis, locally and on Heroku. As we also go through the changelog on each Ruby release...

I also like that it's repeated, I'd rather have the build to break when we upgrade with the wrong version :)

If though we decide to be smart about this, let's go with the all Ruby, without using `cat`

Reply:
FWIW, I don't have the `~>` in my private projects, it's always pinned to a specific version, so I agree with you on that. 

As for whether or not the repetition is good, I can go either way. Personally I prefer having to make a single change and having it automatically being picked up in relevant places, but Ruby version upgrades are infrequent enough for this not to be a big deal.

Reply:
@swrobel could you implement the following changes?

- remove the `~>`
- use Ruby instead of the shell to read the version

than we can merge this ;)